tools/validators/instance_validator/validate/entity_instance.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Uses ontology universe to validate parsed instance data."""
+
+from __future__ import print_function
+from yamlformat.validator import findings_lib
+
+class EntityInstance(findings_lib.Findings):
```

Comment:
assert that the id is not none

Reply:
The check for the `id` key is handled in the syntax parsing, but I'll add a check for redundancy